Sources/BranchSDK/BNCApplication.m

```diff
@@ -69,64 +69,71 @@ + (BNCApplication*) createCurrentApplication {
     return application;
 }
 
-+ (NSDate*) currentBuildDate {
++ (NSDate *)currentBuildDate {
     NSURL *appURL = nil;
     NSURL *bundleURL = [NSBundle mainBundle].bundleURL;
     NSDictionary *info = [NSBundle mainBundle].infoDictionary;
-    NSString *appName = info[(__bridge NSString*)kCFBundleExecutableKey];
+    NSString *appName = info[(__bridge NSString *)kCFBundleExecutableKey];
     if (appName.length > 0 && bundleURL) {
+        // path to the app on device. file:///private/var/containers/Bundle/Application/GUID
         appURL = [bundleURL URLByAppendingPathComponent:appName];
     } else {
+        // TODO: Why is this fallback necessary? The NSBundle approach has been available since iOS 2.0
```

Comment:
Is there a ticket tracking this investigation?

Reply:
Theres a ticket in backlog.